interface/forms_admin/forms_admin.php

```diff
@@ -56,98 +54,66 @@
       sqlQuery("update registry set aco_spec = ? where id = ?", array($val, $matches[1]));
     }
   }
-?>
 
-<?php //ERROR REPORTING
-if ($err)
-	echo "<span class=bold>" . text($err) . "</span><br><br>\n";
-?>
+$viewArgs = [];
+
+//ERROR REPORTING
+if ($err) {
+    $viewArgs['error'] = $err;
+}
+
+//REGISTERED SECTION
+
+// Build registered form array
+foreach($bigdata as $registry) {
+    $priority_category = sqlQuery("SELECT priority, category, nickname, aco_spec FROM registry WHERE id = ?",
+        array($registry['id']));
+    $tmp = [];
+    $tmp['id'] = $registry['id'];
+    $tmp['name'] = $registry['name'];
+
+    if ($registry['sql_run'] == "0") {
+        $tmp['status'] = "Registered";
```

Comment:
How is the getting translated. If this requires translation, and this is just a label, then best to do it here (I think, although I don't have a complete understanding of this code yet; but should soon).